dev/tools/merge-pr.sh

```diff
@@ -18,19 +18,21 @@ else
     QUICK_CONF=""
 fi
 
-RED="\033[31m"
-RESET="\033[0m"
-GREEN="\033[32m"
-BLUE="\033[34m"
-YELLOW="\033[33m"
+# Style terminal codes
```

Comment:
These changes seem completely unrelated to the commit message.

Reply:
@maximedenes : I mentioned in the PR text that I made two rather cosmetic changes on the way - not sure if such cosmetic changes are worthwhile to mention in a commit message.

Reply:
Sorry, I missed it. In principle, a separate commit would have been good (imagine we have to revert the checks because we find out they are too restrictive, we may want to keep the cosmetic changes), but no need to redo it this time.